packages/graphql/src/classes/Node.ts

```diff
@@ -158,9 +186,29 @@ class Node extends GraphElement {
         return this.constrainableFields.filter((field) => field.unique);
     }
 
-    public get plural(): string {
-        const pluralValue = this.nodeDirective?.plural ? this.nodeDirective.plural : pluralize(this.name);
-        return lowerFirst(pluralValue);
+    public get rootTypeFieldNames(): RootTypeFieldNames {
+        const pluralValue = this.plural;
+
+        const pluralValueUpperFirst = upperFirst(pluralValue);
```

Comment:
**suggestion** Just to avoid consistency drift between this and `mutationResponseTypeNames` , consider moving this to a separate private function or getter, used by both methods:

```suggestion
        const pascalValue = this.pascal;
```